plugins/edit/edit.css

```diff
@@ -1,58 +1,43 @@
-div#tedit 
-{
-	width: 405px; 
+div#tedit .row > div {
+  padding: 0;
 }
 
-div#tedit div.dlg-header 
-{
-	background-image: url(../../images/settings.gif)
+div#tedit div.dlg-header {
+  background-image: url(../../images/settings.gif);
 }
 
-div#tedit .text-wrapper
-{
-	border: 1px solid #D0D0D0; 
-	padding: 5px;
-	box-sizing: border-box;
-	-moz-box-sizing: border-box;
-	-webkit-box-sizing: border-box;		
+textarea#etrackers {
+  height: 150px;
+  word-break: break-all;
 }
 
```

Comment:
Could we increase the height of the trackers textarea on desktops, while we're improving the edit dialog? I would like to be able to fit at least 8 trackers (with one line in between) without scrolling.

The line in between is a feature for tracker announce groups. It signals a new announce group for each tracker.